packages/app/utils/formatTimeDate.ts

```diff
@@ -0,0 +1,20 @@
+export function formatTimeDate(dateStr: string) {
+  const date = new Date(dateStr)
+
+  if (Number.isNaN(date.getTime())) {
+    return 'Invalid date'
+  }
+
+  const formattedTime = date
+    .toLocaleTimeString('en-US', { hour: 'numeric', minute: 'numeric', hour12: true })
```

Comment:
this is a good idea, but don't force `en-US` on users. use undefined instead to use their selected locale.

```suggestion
    .toLocaleTimeString(undefined, { hour: 'numeric', minute: 'numeric', hour12: true })
```

Reply:
same for below.